### DISCUSSION TITLE
Lodestar Standup 2025-01 (January 7, 2025)

### DISCUSSION THREAD

**Discussion by philknows** [ChainSafe/lodestar#7326] [Standups]
### Meeting Info
[January 7, 2025, 15:00-16:00 UTC](https://savvytime.com/converter/utc-to-pst-est-cet-ist-ict/jan-7-2025/3pm)
Zoom: shared on #lodestar-general ChainSafe Discord channel at 15:00 UTC

### Agenda
- Open Discussion
  - v1.25.0 planning
  - FOCIL Implementation Planning
- Team Updates
- Closing Remarks

### Instructions
Please post here any topics to be added for Standup discussions. Examples of agenda items are issues, discussions, proposals which require feedback, consensus from the team and/or community members.

**Comment by amlevy327** [comment]
- Discuss SUPRA MoveVM bridge project request: https://discord.com/channels/593655374469660673/605818244036821012/1319742301890941064
-  Over the next few weeks I will be requesting Lodestar feedback from node operators who provided feedback last year. Any specific questions from the team?

**Reply by nflaig** [reply]
> Any specific questions from the team?

I am curious if operators want this feature https://github.com/ChainSafe/lodestar/issues/6509, still not really convinced myself that this is actually useful compared to something like doppelganger protection

**Comment by philknows** [comment]
I'll give a quick update on the BlockOps role closing and make a pitch for a slightly different role that might work better for our needs and potentially find better candidates for.

**Comment by philknows** [comment]
Closing for https://github.com/ChainSafe/lodestar/discussions/7337
Notes are posted here: https://github.com/ChainSafe/lodestar/wiki/Lodestar-Weekly-Standups-2025#lodestar-standup-2025-01-january-7-2025
